ci/terraform/utils/variables.tf

```diff
@@ -111,7 +116,17 @@ variable "bulk_user_email_send_schedule_enabled" {
 
 variable "bulk_user_email_send_schedule_expression" {
   type    = string
-  default = "cron(0 15 ? * MON-FRI *)"
+  default = "cron(0 15 ? * MON-FRI 2049)"
+}
+
+variable "bulk_user_email_audience_loader_schedule_enabled" {
+  type    = bool
+  default = false
+}
+
+variable "bulk_user_email_audience_loader_schedule_expression" {
+  type    = string
+  default = "cron(0 13 ? * MON-FRI 2049)"
```

Comment:
Could be worth adding a description here and on like 118 to summarize the expression

Reply:
designed not to trigger and must be overridden, have added a comment, event is not enabled by default